source/content/drupal-updates.md

```diff
@@ -53,7 +53,7 @@ The critical commands are:
 terminus drush my-drupal-8-site.dev -- migrate-upgrade --legacy-db-key=drupal_7 --configure-only --legacy-root=https://drupal7.example.com
 ```
 
-This command configures (but does not run) the migrations from Drupal 7 to Drupal 8. In this example, the Drupal 8 site is named `my-drupal-8-site` and the command is running on the `dev` environment. The `--legacy-db-key` parameter indicates how to get the login credentials to the source Drupal 7 database. In our example, we use the [Terminus secrets](https://github.com/pantheon-systems/terminus-secrets-plugin) plugin to supply the connection info. [See our blog post for more information on how this flag is used](https://pantheon.io/blog/running-drupal-8-data-migrations-pantheon-through-drush). The `--legacy-root` flag lets Drupal 8 know from where it can grab images and other uploaded media assets.
+This command configures (but does not run) the migrations from Drupal 7 to Drupal 8. In this example, the Drupal 8 site is named `my-drupal-8-site` and the command is running on the `dev` environment. The `--legacy-db-key` parameter indicates how to get the login credentials to the source Drupal 7 database. In our example, we use the [Terminus secrets Manager Plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin) plugin to supply the connection info. [See our blog post for more information on how this flag is used](https://pantheon.io/blog/running-drupal-8-data-migrations-pantheon-through-drush). The `--legacy-root` flag lets Drupal 8 know from where it can grab images and other uploaded media assets.
```

Comment:
The referenced blog post still uses the old secrets plugin (secrets.json). We'll need to either update the blog, or hold off on updating the reference here.

Reply:
The secrets documentation needs to explicitly disambiguate the plugins.  Removing references to the old one alone won't make it go away because customers are familiar with it and still using it.  We need to make clear that there is an old one and a new one and they work differently (and the new one is better).

Reply:
https://github.com/pantheon-systems/documentation/pull/8975/commits/4e22085a98faa25d17b9910226a182d429703e44

Reply:
Updating the blog will be a separate effort. See LOPS-2533